docs/api.rst

```diff
@@ -206,6 +206,51 @@ to handle it, which defaults to print a traceback and ignoring the exception.
     errors. To turn a function into a coroutine, they must be ``async def``
     functions.
 
+.. function:: on_automod_rule_create(rule)
+
+    Called when a new auto moderation rule is created.
+
+    :param rule: The rule that is newly created.
+    :type rule: :class:`AutoModerationRule`
+
+    .. versionadded:: 2.0
+
+.. function:: on_automod_rule_update(before, after)
+
+    Called when an auto moderation rule is updated.
+
+    :param before: The updated rule's old info.
+    :type before: :class:`AutoModerationRule`
+    :param after: The updated rule's new info.
+    :type after: :class:`AutoModerationRule`
+
+    .. versionadded:: 2.0
+
+.. function:: on_automod_rule_delete()
```

Comment:
Whats the point of this event if it doesn't provide the rule that was deleted?

Reply:
it was a docs error not a code error, will add it soon